charts/stable/openvscode-server/questions.yaml

```diff
@@ -7,7 +7,6 @@ portals:
       - "$kubernetes-resource_configmap_portal_host"
     ports:
       - "$kubernetes-resource_configmap_portal_port"
-    path: "/?tkn={{ .Values.env.CONNECTION_TOKEN }}"
 questions:
   - variable: portal
```

Comment:
Look at the portal options in common values.yaml

Reply:
I don't think there is an option for this use case without extending the common lib.

Reply:
I just noticed the portal docs are missing from common. fixing.

Reply:
here is the sneak peak documentation:
```
# -- Set the primary portal for TrueNAS SCALE
portal:
  enabled: false
  # -- Override Path using helm variables
  # path: '/'
  # -- Override the host, for example: by using a custom IP
  # host: 192.168.66.6
  # -- Override the auotmatically gathered ingress port
  # ingressPort: 666
```

Reply:
@stavros-k Just so you know, this code can ALREADY be used.
It already exists, it's just the documentation that was missing, the above is the current defaults.

Reply:
So tried adding this to `values.yaml` (removed the path under questions.yaml)
```
portal:
  path: "/test"
```
But that does not seem to actually work. (It does not override the path)

Reply:
is see that's a bug indeed.